gluoncv/model_zoo/resnet.py

```diff
@@ -97,28 +98,27 @@ def __init__(self, channels, stride, downsample=False, in_channels=0,
             self.se = None
 
         if downsample:
-            self.downsample = nn.HybridSequential(prefix='')
+            self.downsample = nn.HybridSequential()
```

Comment:
reason to remove prefix='' ?

Reply:
Thank you for your review! I am not quite sure but I find that HybridSequential is defined in `nn/basic_layers.py`. In 1.7, its `__init__` function accepts parameters `prefix` and `params` but in master it no longer accepts parameters. It produced erros when I wrote it as `nn.HybridSequential(prefix='')`

Here is a comparison across branches:

![image](https://user-images.githubusercontent.com/32102965/96078198-42253880-0ee4-11eb-9853-96c35892d9e1.png)